data/techs_str.dat

```diff
@@ -268,3 +268,8 @@ result = I can now construct Fuel Cells.
 name = Advanced Fuel Oxidation
 description = My current fuel cell design has room for further improvement.  Extended analysis of new materials and designs may let me overcome the limits of my previous efforts, but such research involves dangerous chemicals and potential reactions that may be best handled far away from population centers.
 result = I can now construct Solid Fuel Cells.
+
+[Knownledge Preservation]
+name = Knownledge Preservation
+description = The preservation of knowledge is a major concern for humanity against the ravages of time and catastrophic failure. I should use my new capababilies and informations to review their solutions and adapt it to hold my conscience.
```

Comment:
Here's the rewrite:

"Humanity expends a great deal of effort on maintaining its history and the knowledge of various technologies that its cultures have come to rely on, even in the case of major catastrophic events.  Perhaps researching their methodologies will provide me with similar ideas for long-term safeguards to help guarantee my continued existence."

Reply:
Thx !